kas-rusty-kaspad/umbrel-app.yml

```
@@ -0,0 +1,31 @@
+manifestVersion: 1
+id: kas-rusty-kaspad
```

Would this id not fit better? If so, please rename it and also the folder.
```suggestion
id: rusty-kaspad
```